src/bluefin_v2_client/client.py

```diff
@@ -392,14 +393,17 @@ async def withdraw_margin_from_bank(self, amount: Union[float, int]) -> bool:
             "margin_bank",
             self.account.getUserAddress(),
             self.contracts.get_package_id(),
+            typeArguments=[self.contracts.get_currency_type()],
         )
         signature = self.contract_signer.sign_tx(txBytes, self.account)
         res = rpc_sui_executeTransactionBlock(self.url, txBytes, signature)
-
-        if res["result"]["effects"]["status"]["status"] == "success":
-            return True
-        else:
-            return False
+        try:
+            if res["result"]["effects"]["status"]["status"] == "success":
+                return True
+            else:
+                return False
+        except Exception as e:
```

Comment:
Why do we return in res object in case of exception instead of error ?

Reply:
since its kind of like a api call, its better to just give the user the result so that he can identify the issue. it can be 404 or many other possibilities